redirects.yaml

```diff
@@ -506,6 +506,12 @@ recaptimewiki:
   ca: 'Sectigo'
   hsts: 'strict'
   disable_event: true
+wikirtapptk:
```

Comment:
This isn't an *.miraheze.org redirect, so it should be moved above to be under irishfreemaninfo.com.

Reply:
Thanks for the note, editing right now.